packages/ui/src/ui/containers/CellPreviewModal/CellPreviewModal.tsx

```diff
@@ -47,24 +48,30 @@ export const CellPreviewModal: React.FC = () => {
             <Flex className={b('content')} gap={2} direction="column">
                 <Flex gap={2} direction="column">
                     <Text variant="subheader-3" color="secondary">
-                        {data?.$incomplete && !error
-                            ? 'Unable to load content more than 16MiB. Please use the command bellow to\n' +
-                              '                            load it locally.'
-                            : 'You could use the command bellow to load it locally.'}
+                        {incompleteNoticeText && incompleteNoticeText}
```

Comment:
Looks like a misprint:
```ts
{incompleteNoticeText && incompleteNoticeText}
// it looks like it should be replaced with
{incompleteNoticeText}
// if so the field should be renamed to `noticeText`
// cause we don't need to check `data?.$incomplete` here
```

Reply:
Yes, it looks like it is better to replace it with noticeText and set the text itself in the action. Then we can put all the checks into action